webpack.config.js

```diff
@@ -65,11 +67,17 @@ const JS6_FILES = [
     './wyoming/index.js6',
 ];
 
+const JS6_FILES = [

```

Comment:
Split these up only so I can pass just one of the state's JS files to eslint. Happy to do it differently if this seems odd to anyone.